Singular/LIB/sresext.lib

```diff
@@ -0,0 +1,774 @@
+//////////////////////////////////////////////////////////////////////////////
+version=" ";
+category="Noncommutative";
+info="
+LIBRARY: sresforextalg.lib     Schreyer's algorithm for modules over the exterior algebra
+AUTHORS: Ramesh, L.,           ramesh@mathematik.uni-kl.de   
+
+OVERVIEW:
+Procedures for and involving computing Schreyer resolution of modules over the exterior algebra
+
+PROCEDURES:
+schreyerSyzExt(m);            (refined) Schreyer syzygy of a module over the exterior algebra 
+sresExt(m,p);                 free resolution of module m over the exterior algebra using refined Schreyer alg        
+mresExt(m,p);                 minimal free resolution of module m over the exterior algebra using refined Schreyer alg
+sheafCohBGGsres(M,l,h)        cohomology of twists of the coherent sheaf F on P^n
+                              associated to coker(M). The range of twists is determined by @code{l},
+                              @code{h}.
+";
+
+static proc mod_init()
```

Comment:
please change the proc mod_init to
LIB "nctools.lib"; // for ncalg
LIB "matrix.lib"; // for submat
LIB "sheafcoh.lib"; // for displayCohom
```